ciri/api/api.h

```diff
@@ -23,8 +24,10 @@ typedef struct iota_api_s {
   thread_handle_t thread;
   bool running;
   uint16_t port;
+  tangle_t *tangle;
   serializer_t serializer;
   serializer_type_t serializer_type;
+  size_t max_get_trytes;
```

Comment:
could we encode this in some `api_limits_t` struct? there's also other limits on the API